### ISSUE TITLE
Use TSV

### ISSUE THREAD

**Issue by GrimPixel** [emilepharand/Babilonia#381]
The DB file is not that easy to use. Maybe you can consider TSV?

**Comment by emilepharand** [comment]
Hi GrimPixel! Thank you for your suggestion.

The database is a SQLite file meant to be used by the application. You can use [DB Browser for SQLite](https://sqlitebrowser.org/dl/) if you would like to navigate it. You can also find tools to convert from SQL to TSV.

While a TSV file could be more accessible and easier to manipulate for some users, it would not support the CRUD operations and data relationships that our application needs. SQL is well-suited for these requirements, which is why we've chosen to use it.

That being said, I agree that it would be a great idea for the base database to be available as a text-based format such as TSV, which could then be imported into the application. When we do #7, we should certainly consider that.

**Comment by GrimPixel** [comment]
Yes. I understand now. It's just  a different style from mine. I tend to write scripts to do the CRUD works.
